SwitchCalculator.Task3/src/MySwitchCalcTest3.java

```diff
@@ -0,0 +1,37 @@
+
+import org.w3c.dom.ls.LSOutput;
+
+import java.util.Scanner;
+public class MySwitchCalcTest3 {
+    public static void main(String[] args) {
+        while (true) {
```

Comment:
Please change while(true) to more correct one. Maybe another case for exit from program